.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: c
+dist: trusty
+
+cache:
+  directories:
+    - ~/docker
+
+matrix:
+  include:
+  - env: BASE=ubuntu IMAGE=ubuntu:16.04 CCO=gcc
+
+  - env: BASE=ubuntu IMAGE=ubuntu:16.04 CCO=clang
+
+  - env: BASE=ubuntu IMAGE=ubuntu:16.10 CCO=gcc
+
+  - env: BASE=fedora IMAGE=fedora:25    CCO=gcc
+
+before_install:
+  - cd ..
+  - git clone --depth 1 https://github.com/GNOME/jhbuild.git
+
+before_script:
+  - export NAME=$(echo $IMAGE | sed "s/:/_/g")
+  - ls -lah ~/docker
```

Comment:
Yes, I already committed (on this branch) a fix, but since it is easy to deal with incremental patches from now on, I haven't pushed.